iothub/device/samples/PnpDeviceSamples/TemperatureController/Parameter.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using CommandLine;
+
+namespace TemperatureController
+{
+    /// <summary>
+    /// Parameters for the application supplied via command line arguments.
+    /// If the parameter is not supplied via command line args, it will look for it in environment variables.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            's',
+            "DeviceSecurityType",
+            HelpText = "(Required) The provisioning type that will be used for provisioning the device for the sample. Possible values include dps, connectionString (case-insensitive)." +
```

Comment:
...that will be used for **connecting** the device...

Otherwise we use the word provisioning twice, which sounds weird. Also, if I use a connection string to the hub, it isn't really provisioning. Might think that through a bit.

Reply:
Yeah, I am still looking for the proper term to use here - it isn't really a security type, is it?
I do like connection here - maybe we can rename it to "deviceConnectionType" and use "dps" and "iothub" as possible values? Although, even if you connect _via_ dps, you will still connect to iothub finally.